### PR TITLE
Add semantic-release settings

### PR DESCRIPTION
**Pull request by relekang**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,8 +1,17 @@
+import re
 import os
+
 from setuptools import setup, find_packages
 
 os.environ['DJANGO_SETTINGS_MODULE'] = 'tests.settings'
 
+with open('frigg_settings/__init__.py', 'r') as fd:
```
Comment (ekmartin) [inline setup.py]: frigg_settings? :D

Comment (relekang) [inline setup.py]: Copy paste ftw :p

Comment (relekang) [inline setup.py]: Fixed it

**Comment by ekmartin** [comment]
Nice :+1:

**Pull request merged**